worker/downloader.go

```diff
@@ -44,8 +44,8 @@ func newDownloader(ctx context.Context, host Host) *downloader {
 	return &downloader{
 		host: host,
 
-		statsSectorDownloadEstimateInMS: stats.Default(),
-		statsDownloadSpeedBytesPerMS:    stats.NoDecay(),
+		statsSectorDownloadEstimateInMS: utils.NewDataPoints(10 * time.Minute),
```

Comment:
I considered keeping two constructors and/or exposing a constant with the default decay, but went with writing it in full in the end because I like the documentation aspect. In time we'll get rid of this anyway, fine with changing it though I went back and fourth.